### ISSUE TITLE
Icon is blank

### ISSUE THREAD

**Issue by GFialho** [0xsequence/rainbowkit-plugin#9]
Icon is not showing on rainbowkit:
![image](https://github.com/0xsequence/rainbowkit-plugin/assets/108136415/2405ce08-c969-4971-8b7e-3f695e1832dd)

You can check on the live demo page: https://0xsequence.github.io/demo-dapp-rainbowkit/

**Comment by pkieltyka** [comment]
thanks for the heads up -- we'll fix this in a few min :)

**Comment by SebastianJ21** [comment]
Was just about to submit a PR for you :)

The issue is just a missing semicolon in the `const icon`:

```
`data:image/svg+xmlbase64,` -> `data:image/svg+xml;base64,`
```

**Comment by pkieltyka** [comment]
thanks :)

**Comment by pkieltyka** [comment]
resolved in v2.1.4 :)
